src/tron.jl

```diff
@@ -109,31 +113,30 @@ function TronSolver(nlp::AbstractNLPModel{T, V};) where {T, V <: AbstractVector{
   Hs = V(undef, nvar)
   H = hess_op!(nlp, x, Hs)
   Op = typeof(H)
-  tr = TrustRegion(gt, one(T))
+  tr = TRONTrustRegion(gt, min(one(T), max_radius - eps(T)); max_radius = max_radius, kwargs...)
   return TronSolver{T, V, Op}(x, xc, temp, gx, gt, gn, gpx, Hs, H, tr)
 end
 
 function SolverCore.reset!(solver::TronSolver)
   solver.tr.good_grad = false
-  solver.tr.radius = solver.tr.initial_radius
```

Comment:
Why remove this?

Reply:
Because, it is done in the `solve!` function now